metashare/api/models.py

```diff
@@ -540,6 +545,10 @@ def save(self, *args, **kwargs):
 
         return ret
 
+    def mark_visited(self):
+        self.has_ever_been_visited = True
+        self.save()
```

Comment:
@wlonk I think we also need to set this back to `False` when the org is refreshed?

Reply:
So it's not "ever" then? It's "visited since last refresh"?

Reply:
Yes, I think so. A refresh is essentially a new org.